[skip travis] Add more content to embed README.md (#1931)
Microsoft/BotFramework-WebChat: README.md

  <a href="https://coveralls.io/github/Microsoft/BotFramework-WebChat?branch=master"><img src="https://coveralls.io/repos/github/Microsoft/BotFramework-WebChat/badge.svg?branch=master" alt="Coverage Status" /></a>
</p>

## Migrating from Web Chat v3 to v4

There are three possible paths that migration might take when migrating from v3 to v4. First, please compare your beginning scenario:

### My current website integrate Web Chat using an `<iframe>` element obtained from Azure Bot Services. I want to upgrade to v4.

> Starting from May 2019, we are rolling out v4 to websites that integrate Web Chat using `<iframe>` element. Please refer to [this article](https://github.com/Microsoft/BotFramework-WebChat/treemaster/packages/embed/README.md) for information on integrating Web Chat using `<iframe>`.

Reviewer: ```suggestion
Starting from May 2019, we are rolling out v4 to websites that integrate Web Chat using `<iframe>` element. Please refer to [the embed documentation](https://github.com/Microsoft/BotFramework-WebChat/treemaster/packages/embed/README.md) for information on integrating Web Chat using `<iframe>`.
```

Don't forget that saying 'this documentation' is not good accessibility, since it doesn't tell the user what they will be clicking.
